tutorial.json

```diff
@@ -97,12 +94,16 @@
             "You should finish your project while on the `main` branch. Your working tree should be clean and you should not have any uncommitted changes"
           ],
           "setup": {
-            "commits": [
-              "e3658057cb2e1f3a903158fcdd3c8be2dc1ed579"
-            ]
+            "commits": []
           }
         }
-      ]
+      ],
+      "setup": {
+        "commits": [
+          "529ca8c4d6a5138675ff46303bcc1767d62b8bf5",
```

Comment:
These commit hashes should not have changed. You didn't do something correct here. Likely that you don't have the correct `v1.0.3` branch locally. Try doing this: 
- Delete your local `v1.0.3` branch if it exists: `git branch -D v1.0.3`
- Make sure you have a remote to this repo: `git remote add upstream https://github.com/freeCodeCamp/learn-periodic-table-database.git` if you don't
- Make sure you have all the remotes updated: `git fetch --all`
- Re-create the `v.1.0.3` branch: `git branch v1.0.3 upstream`
- Make sure you are on the `main` branch and rebuild the `tutorial.json` file: `coderoad build`

Reply:
That's why me pr looked different, on it 👍

Reply:
I made new PR #12 , I should have update it in this branch after updating the json. sorry about that